libobs/obs-properties.c

```diff
@@ -271,6 +271,30 @@ obs_property_t *obs_properties_get(obs_properties_t *props, const char *name)
 	return NULL;
 }
 
+void obs_properties_remove(obs_properties_t * props, const char *name)
```

Comment:
Style:  Place asterisk next to parameter name.

```suggestion
void obs_properties_remove(obs_properties_t *props, const char *name)
```